configs/DAKEFPVH743_SLIM/config.h

```diff
@@ -0,0 +1,190 @@
+#define SPI2_SDI_PIN         PC2 
+#define SPI2_SDO_PIN         PC3 
+#define SPI3_SCK_PIN         PC10
+#define SPI3_SDI_PIN         PC11
+#define SPI3_SDO_PIN         PC12
+#define SPI4_SCK_PIN         PE12
+#define SPI4_SDI_PIN         PE5
+#define SPI4_SDO_PIN         PE6
+#define ADC_VBAT_PIN         PA4 
+#define ADC_RSSI_PIN         PC5 
+#define ADC_CURR_PIN         PC0
+#define ADC_AIRS_PIN         PC4 
+#define ADC_EXTERNAL1_PIN    ADC_AIRS_PIN
+#define PINIO1_PIN           PE2
+#define PINIO2_PIN           PE3
+#define PINIO3_PIN           PE4
+#define PINIO4_PIN           PD2
+#define FLASH_CS_PIN         PA15
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PD4 
+#define GYRO_1_CS_PIN        PC9
+#define GYRO_2_EXTI_PIN      PB2
+#define GYRO_2_CS_PIN        PB1
+
+#define USE_GYRO_CLKIN
+#define GYRO_1_CLKIN_PIN     PB7 // TIM17_CH1N
+
+#define SDIO_CK_PIN          PC1  
+#define SDIO_CMD_PIN         PD7  
+#define SDIO_D0_PIN          PB14 
+#define SDIO_D1_PIN          PB15 
+#define SDIO_D2_PIN          PB3  
+#define SDIO_D3_PIN          PB4  
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0,  PE9,  1,  0) \
+    TIMER_PIN_MAP( 1,  PE11, 1,  1) \
+    TIMER_PIN_MAP( 2,  PE13, 1,  2) \
+    TIMER_PIN_MAP( 3,  PE14, 1,  3) \
+    TIMER_PIN_MAP( 4,  PA0,  1,  4) \
+    TIMER_PIN_MAP( 5,  PA1,  1,  5) \
+    TIMER_PIN_MAP( 6,  PA2,  1,  6) \
+    TIMER_PIN_MAP( 7,  PA3,  1,  7) \
+    TIMER_PIN_MAP( 8,  PD12, 1, -1) \
+    TIMER_PIN_MAP( 9,  PD13, 1, -1) \
+    TIMER_PIN_MAP( 10, PD14, 1, -1) \
+    TIMER_PIN_MAP( 11, PD15, 1, -1) \
+    TIMER_PIN_MAP( 12, PC8,  2, -1) \
+    TIMER_PIN_MAP( 13, PB7,  1, -1) \
+    TIMER_PIN_MAP( 14, PB0 , 2,  8)
+
+#define ADC1_DMA_OPT        9
+#define ADC3_DMA_OPT        10
+
+#define TIMUP1_DMA_OPT      11
+#define TIMUP2_DMA_OPT      12
+
+#define FLASH_SPI_INSTANCE SPI3
+#define MAX7456_SPI_INSTANCE SPI2
+
+#define BARO_I2C_INSTANCE I2CDEV_2
+#define MAG_I2C_INSTANCE I2CDEV_2
+
+#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_SDCARD
+#define DEFAULT_DSHOT_BITBANG DSHOT_BITBANG_ON
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_GYRO_TO_USE GYRO_CONFIG_USE_GYRO_BOTH
+
+#define SDCARD_DETECT_PIN NONE
+#define SDIO_DEVICE SDIODEV_2
+#define SDIO_USE_4BIT 1
+
+#define BEEPER_INVERTED
+#define PINIO1_CONFIG 129
+#define PINIO1_BOX 40
+#define BOX_USER1_NAME                      "CAM 1,2"
+#define PINIO2_CONFIG 129
+#define PINIO2_BOX 41
+#define BOX_USER2_NAME                      "12V BEC"
+#define PINIO3_CONFIG 129
+#define PINIO3_BOX 42
+#define BOX_USER3_NAME                      "RF SWITCH"
+#define PINIO4_CONFIG 129
+#define PINIO4_BOX 0
+#define BOX_USER4_NAME                      "CAN SILENT"
+
+#define GYRO_1_SPI_INSTANCE SPI1
+#define GYRO_1_ALIGN CW0_DEG_FLIP
+#define GYRO_2_SPI_INSTANCE SPI4
+#define GYRO_2_ALIGN CW90_DEG_FLIP
+
+#define DEFAULT_CURRENT_METER_SCALE 120  
```

Comment:
Determined by the ESC. Should not be set unless the FC is an AIO or only sold as a stack with ESC.
```suggestion

```

Reply:
Modified